hathor/util.py

```diff
@@ -759,3 +763,33 @@ def is_token_uid_valid(token_uid: bytes) -> bool:
         return True
     else:
         return False
+
+
+@dataclass
+class EnvironmentInfo:
+    # Changing these names could impact logging collectors that parse them
+    python_implementation: str
+    hathor_core_args: str
+    hathor_core_version: str
+    peer_id: Optional[str]
+    network: str
+
+    def as_dict(self):
+        return asdict(self)
+
+
+def get_environment_info(args: Namespace, peer_id: Optional[str]) -> EnvironmentInfo:
```

Comment:
I feel `Namespace` is more related to argparse. Maybe this method should receive the `args` as a string to make it more adaptable to other cases.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/490/commits/cd78536cfae9fc4d4335f4d7838415a06867e428

Reply:
Thanks!